x/crosschain/types/revert_options.go

```diff
@@ -49,6 +50,21 @@ func NewRevertOptionsFromEVM(revertOptions gatewayevm.RevertOptions) RevertOptio
 	}
 }
 
+// NewRevertOptionsFromSOL initializes a new RevertOptions from a solana.RevertOptions
+func NewRevertOptionsFromSOL(revertOptions solana.RevertOptions) RevertOptions {
```

Comment:
As per the Solana pr, I suggest to add onAbort address parsing here.

By adding it, the onAbort workflow should be automatically supported for Solana